warden/setup-environment/action.yml

```
@@ -0,0 +1,398 @@
+name: "Setup Mage-OS in Warden"
+author: "Vladyslav Podorozhnyi"
```

```suggestion
author: "Mage-OS"
```

Should we remove this or make this Mage-OS? 🙈